pygsp2/utils_examples.py

```diff
@@ -20,8 +25,8 @@
 import numpy as np
 
 
-def make_metro_graph(edgesfile='santiago_metro_stations_connections.txt',
-                     coordsfile='santiago_metro_stations_coords.geojson'):
+def make_metro_graph(edgesfile='../pygsp2/data/santiago_metro_stations_connections.txt',
+                     coordsfile='../pygsp2/data/santiago_metro_stations_coords.geojson'):
```

Comment:
Why this change here?

Reply:
The original function in "utils_examples.py" was hard coded/ bound to data contained in the examples folder, which was not accessible from within the function, easiest way to solve it was to move said files to the data folder of PyGSP.